### PR TITLE
Update dg

### PR DESCRIPTION
**Pull request by HollaG**

### CODE REVIEW

**Comment by HollaG** [comment]
do not merge yet

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ The sections below give more details of each component.
 
 ### UI component
 
-The **API** of this component is specified in [`Ui.java`](https://github.com/AY2324S1-CS2103T-T11-3/tp/blob/master/src/main/java/seedu/address/ui/Ui.java)
+The **API** of this component is specified in [`UI.java`](https://github.com/AY2324S1-CS2103T-T11-3/tp/blob/master/src/main/java/seedu/address/ui/UI.java)
```
Comment (pzl111) [inline docs/DeveloperGuide.md]: uh not really i think

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -141,25 +141,27 @@ The sequence diagram below illustrates the interactions within the `Logic` compo
...
-The `Command` class will do the actual modification of the data, when executed by the LogicManager, and communicate its execution result via the CommandResult class with the Logic manager. 
+The `Command` class will do the actual modification of the data, when executed by the `LogicManager`, and communicate its execution result via the `CommandResult` class with the `LogicManager`. 
 
 Parser classes are responsible for parsing the user input and creating the corresponding command object. 
 
-AddressBookParser is responsible for parsing the user input for finding the corresponding parser and returning the corresponding command object. 
+`AddressBookParser` is responsible for parsing the user input for finding the corresponding parser and returning the corresponding command object. 
 
-LogicManager will perform the actual execution of the command, and update the ui and storage.
+`LogicManager` will perform the actual execution of the command, and update the UI and Storage.
```
Comment (pzl111) [inline docs/DeveloperGuide.md]: i think these are okay, since UI refer to user interface not the Ui class

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -878,7 +880,7 @@ testers are expected to do more *exploratory* testing.
     * The response box is to display responses for command execution, to indicate success or errors.
     * The left side has the list panel, which shows different list types (student, schedule, tasks).
     * The right side has the details panel, which shows details of any specific item in the list.
-      ![Ui](images/Ui.png)
+      ![UI](images/UI.png)
```
Comment (pzl111) [inline docs/DeveloperGuide.md]: might be broken link

**Review by pzl111 — CHANGES_REQUESTED** [review]
some comments

**Review by elaineshijie — APPROVED** [review]
LGTM!

**Review by pzl111 — APPROVED** [review]
LGTM

**Pull request merged**